<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnminimumwidth="2,1">
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_addresult"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_a"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_a">
      <property name="text">
       <string>a的值(double)：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_b"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_addresult">
      <property name="text">
       <string>a+b的值(double)：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_b">
      <property name="text">
       <string>b的值(double)：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>计算a+b</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
